libr/debug/p/native/linux/linux_coredump.c

```diff
@@ -44,7 +44,7 @@ static bool is_a_kernel_mapping(const char *name) {
 		&& strcmp (name, "[vvar]")
 		&& strcmp (name, "[heap]")
 		&& strcmp (name, "[vectors]")
-		&& strncmp (name, "[stack", strlen ("[stack")));
+		&& strncmp (name, "[stack", sizeof ("[stack") - 1));
```

Comment:
sizeof??

Reply:
`        size_t a = sizeof("[stack") - 1;
        size_t b = strlen("[stack");
        printf("%zu %zu\n", a, b);`

Both give 6.

Reply:
yeah, but sizeof will be affected by the encoding of the string and we use strlen for this case

Reply:
Sorry I forgot to change it back...